kano_greeter/user_list.py

```diff
@@ -70,32 +75,19 @@ def add_account(*_):
 
 
 
-class User(Gtk.EventBox):
+class User(KanoButton):
     HEIGHT = 50
 
     def __init__(self, username):
-        Gtk.EventBox.__init__(self)
+        KanoButton.__init__(self, text=username.title(), color="orange")
```

Comment:
True. Are there any sane pythonesque rules for when to choose which or do you guys have an internal rule? I'm from the Perl world, where the quotes have different meanings, and [people seem to agree](https://www.python.org/dev/peps/pep-0008/#string-quotes) that the following is a good rule (at least it makes sense to me, too):

> I like to use double quotes around strings that are used for interpolation or that are natural language messages, and single quotes for small symbol-like strings.

On the other hand, the Python guys themselves [don't recommend one over the other](https://www.python.org/dev/peps/pep-0008/#string-quotes).